src/components/header/loginButton.tsx

```diff
@@ -56,11 +56,11 @@ const LoginButton = () => {
               key="profile"
               placement="bottom-end"
               rootClose
-              show={showPopover}
               overlay={
                 <Popover
                   id={"ProfilePopover"}
                   className="border border-2 border-tertiary"
```

Comment:
What is moving the show attribute here meant to achieve?

Reply:
The show attribute in the OverlayTrigger breaks the rootClose. The show attribute doesn't really work at all in any case.